doc/Hammer-Use/index.rst

```diff
@@ -0,0 +1,13 @@
+Hammer Use
+========================================
+
+This documentation will walk through the currently supported steps of the Hammer flow including synthesis, place-and-route, DRC, LVS, simulation, and more.
```

Comment:
I don't know what this is supposed to say, but this seems incorrect based on the sections below. Is this a copy/paste bug?